### PR TITLE
fixed warns in automated linter

### PR DESCRIPTION
**Pull request by jsantorek**

### CODE REVIEW

**Comment by jsantorek** [comment]
fixes these warnings:
image

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4.2.2
```
Comment (noomorph) [inline .github/workflows/ci.yml:27]: Why do you use specific version, not just v4 ?

Comment (jsantorek) [reply .github/workflows/ci.yml:27]: no particular reason - usually I just explicitly grab the latest when updating. Do github actions work with semantic versioning - aka, doing `@v4` will grab latest available?

Comment (jsantorek) [reply .github/workflows/ci.yml]: yup, just confirmed that they seem to be re-tagging repo

**Review by noomorph — APPROVED** [review]

**Comment by noomorph** [comment]
Thanks!

**Pull request merged**